lib/fake_artist_web/channels/table_channel.ex

```diff
@@ -8,6 +8,7 @@ defmodule FakeArtistWeb.TableChannel do
 
   def handle_info({:after_join, table_name}, socket) do
     table_pid = FakeArtist.Hostess.get_table_pid(table_name)
+    FakeArtist.Table.add_player(table_pid, socket.channel_pid)
```

Comment:
like my previous comment, don't pass the channel pid explicitly, just do `FakeArtist.Table.add_self(table_pid)` or w.e you want to call it